libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/StreamingExtensions/BotFrameworkWebSocketAdapter.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Protocol.WebSockets;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Bot.Builder.Integration.AspNet.Core.StreamingExtensions
+{
+    public class BotFrameworkWebSocketAdapter : IBotFrameworkHttpAdapter
+    {
+        private readonly IChannelProvider _channelProvider;
+        private readonly ICredentialProvider _credentialProvider;
+
+        public BotFrameworkWebSocketAdapter(ICredentialProvider credentialProvider, IChannelProvider channelProvider = null, ILogger<BotFrameworkWebSocketAdapter> logger = null)
+        {
+            this._credentialProvider = credentialProvider;
+            this._channelProvider = channelProvider;
+        }
+
+        public ConcurrentDictionary<string, WebSocketServer> Connections { get; set; }
+
+        public async Task ProcessAsync(HttpRequest httpRequest, HttpResponse httpResponse, IBot bot, CancellationToken cancellationToken = default(CancellationToken))
+         {
+            if (httpRequest == null)
+            {
+                throw new ArgumentNullException(nameof(httpRequest));
+            }
+
+            if (httpResponse == null)
+            {
+                throw new ArgumentNullException(nameof(httpResponse));
+            }
+
+            if (bot == null)
+            {
+                throw new ArgumentNullException(nameof(bot));
+            }
+
+            var authHeader = httpRequest.Headers["Authorization"];
+            var channelId = httpRequest.Headers["ChannelId"];
+            try
+            {
+                var claimsIdentity = await JwtTokenValidation.ValidateAuthHeader(authHeader, _credentialProvider, _channelProvider, channelId).ConfigureAwait(false);
+                if (!claimsIdentity.IsAuthenticated)
+                {
+                    httpRequest.HttpContext.Response.StatusCode = (int)HttpStatusCode.Unauthorized;
+                    return;
+                }
+            }
+            catch (Exception)
```

Comment:
>Exception [](start = 19, length = 9)

Since we are not using the Exception , we can use the empty catch { } syntax

Reply:
This is me being lazy and wanting it to be easy to search for 'Exception' after a talk with the SDK team on what I'm allowed to log here. My inclination is we SHOULD be doing something with them, but the existing code doesn't, so I want to know the reasoning before adding logging/handing to ours.